cluster/pulumi/infra/grafana-dashboards/canton-network/catchup.json

```diff
@@ -350,7 +350,7 @@
           },
           "disableTextWrap": false,
           "editorMode": "code",
-          "expr": "time() - max by (namespace) ((daml_sequencer_client_handler_last_sequencing_time_micros{namespace=~\"$namespace\",component=\"participant\"} != 0) /1e6)",
+          "expr": "max by (namespace) (timestamp(daml_sequencer_client_handler_last_sequencing_time_micros{namespace=~\"$namespace\",component=\"participant\"}) - ((daml_sequencer_client_handler_last_sequencing_time_micros{namespace=~\"$namespace\",component=\"participant\"} > 0) / 1e6))",
```

Comment:
So0removing the filter for the first one works because it does essentially a join for the subtraction on labels so if there is nothing in the second it also produces no entries?

Reply:
yup, and it sort of makes sense because the timestamp function gives you the time of the series you pass, if you add a filter you get a new series it seems so you get the time of the evaluation of the filtering